src/config/localization/translations.json

```diff
@@ -1429,5 +1429,26 @@
   "You have %amount% ticket this round": "You have %amount% ticket this round",
   "You had %amount% tickets this round": "You had %amount% tickets this round",
   "You had %amount% ticket this round": "You had %amount% ticket this round",
-  "Remove Liquidity": "Remove Liquidity"
+  "Remove Liquidity": "Remove Liquidity",
+  "Minimum received": "Minimum received",
+  "Maximum sold": "Maximum sold",
+  "The difference between the market price and estimated price due to trade size.": "The difference between the market price and estimated price due to trade size.",
+  "The difference between the market price and your price due to trade size.": "The difference between the market price and your price due to trade size.",
+  "Liquidity Provider Fee": "Liquidity Provider Fee",
+  "Import Token": "Import Token",
+  "Output is estimated. You will receive at least %amount% %symbol% or the transaction will revert.": "Output is estimated. You will receive at least %amount% %symbol% or the transaction will revert.",
+  "Input is estimated. You will sell at most %amount% %symbol% or the transaction will revert.": "Input is estimated. You will sell at most %amount% %symbol% or the transaction will revert.",
+  "Output will be sent to %recipient%": "Output will be sent to %recipient%",
```

Comment:
Wait, that feature still exists? Though we did remove it @RabbitDoge

Reply:
Yes, it shows when you enable the expert mode